appveyor.yml

```diff
@@ -37,7 +38,7 @@ install:
 build: false  # Not a C# project, build stuff at the test step instead.
 
 test_script:
-  # Run the project tests, but (sadly) exclude ones that take a long time
+  # Run the project tests, but (sadly) exclude the ones that take a long time
```

Comment:
What about this comment ... is this a copy paste remnant ... or is this relevant and should stay?

otherwise, I'd delete it because I cannot see how it makes sense

Reply:
lol, more likely copy-paste ...